.github/pull_request_template.md

```diff
@@ -26,22 +26,23 @@ Soundtrack of this PR: [link to song that really fits the mood of this PR]()
 | <please always include, at least, a min and max screen> |        |       |
 
 ### Testing
+
 < Copy and paste the results of `yarn test test/dir/files --coverage`>
 
 #### Example (Delete this):
 
- PASS  test/fullService/api/getLockedStatus.test.ts
- PASS  test/fullService/axiosFullService.test.ts
-File                  | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
+PASS test/fullService/api/getLockedStatus.test.ts
+PASS test/fullService/axiosFullService.test.ts
+File | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
 ----------------------|---------|----------|---------|---------|-------------------
-All files             |     100 |      100 |     100 |     100 |
- fullService          |     100 |      100 |     100 |     100 |
-  axiosFullService.ts |     100 |      100 |     100 |     100 |
- fullService/api      |     100 |      100 |     100 |     100 |
-  getLockedStatus.ts  |     100 |      100 |     100 |     100 |
-  index.ts            |       0 |        0 |       0 |       0 |
+All files | 100 | 100 | 100 | 100 |
+fullService | 100 | 100 | 100 | 100 |
+axiosFullService.ts | 100 | 100 | 100 | 100 |
+fullService/api | 100 | 100 | 100 | 100 |
+getLockedStatus.ts | 100 | 100 | 100 | 100 |
+index.ts | 0 | 0 | 0 | 0 |
```

Comment:
Why this change? I think this file should be removed from the PR.

Reply:
I made an edit to the original when starting this branch with gitflow to try and have it use the info I put into the template, didn't work so this it to restore it to its original state.